### PR TITLE
fix(wallet): Fix recovery phrase spacing and wrapping

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves [YOMO-1148](https://emurgo.atlassian.net/browse/YOMO-1148)

[YOMO-1148]: https://emurgo.atlassian.net/browse/YOMO-1148?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by stackchain — APPROVED** [review]

File: apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx [hunk 1]
```diff
@@ -206,17 +207,17 @@ const styles = StyleSheet.create({
     paddingHorizontal: 16,
   },
   recoveryPhrase: {
-    height: 26 * 7,
-    paddingHorizontal: 16,
-  },
-  recoveryPhraseOutline: {
-    flex: 1,
+    marginHorizontal: 16,
```
Comment (stackchain) [inline apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: @michaeljscript can we replace with `<Spacer s?`

Comment (stackchain) [reply apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: margin is evil :D

Comment (michalszorad) [reply apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: Having a look

Comment (michalszorad) [reply apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: I've removed the margin and did some refactoring

File: apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx [hunk 2]
```diff
@@ -197,33 +185,26 @@ const styles = StyleSheet.create({
...
-    paddingHorizontal: 16,
-  },
   recoveryPhrase: {
-    height: 26 * 7,
-    paddingHorizontal: 16,
-  },
-  recoveryPhraseOutline: {
-    flex: 1,
+    padding: 8,
+    minHeight: 26 * 7,
```
Comment (jorbuedo) [inline apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: It was already there, but is there a reason for this operation?

**Review by jorbuedo — APPROVED** [review]

File: apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx (hunk 2 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: I think it was meant to be 26px * 7 lines, but looking at the UI, this does not match 7 lines. Will change it to 182, so it does not create confusion

File: apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx [hunk 3]
```diff
@@ -111,18 +102,14 @@ const MnemonicInput = ({userEntries, error, onPress}: MnemonicInputProps) => {
...
-  )
+  return <Text>{strings.instructions}</Text>
 }
 
 const ErrorMessage = ({visible}: {visible: boolean}) => {
   const strings = useStrings()
 
   return (
-    <View style={[styles.error, !visible && styles.hidden]}>
+    <View style={[!visible && styles.hidden]}>
```
Comment (banklesss) [inline apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]:
```suggestion
    <View style={!visible && styles.hidden}>
```

Comment (michalszorad) [reply apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx]: Updated

**Review by banklesss — APPROVED** [review]

**Review by stackchain — APPROVED** [review]

**Pull request merged**